src/nft/components/profile/list/RoyaltyTooltip.tsx

```diff
@@ -1,27 +1,87 @@
 import { Trans } from '@lingui/macro'
-import { ListingMarket } from 'nft/types'
-// eslint-disable-next-line no-restricted-imports
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { ListingMarket, WalletAsset } from 'nft/types'
+import { formatEth } from 'nft/utils'
 import styled from 'styled-components/macro'
 import { ThemedText } from 'theme'
 
-const FeeWrap = styled.div`
+const FeeWrap = styled(Row)`
   margin-bottom: 4px;
-  color: ${({ theme }) => theme.textPrimary};
+  justify-content: space-between;
 `
 
-const RoyaltyContainer = styled(ThemedText.BodySmall)`
-  margin-bottom: 8px;
+const RoyaltyContainer = styled(Column)`
+  gap: 12px;
+  padding 4px 0px;
 `
 
-export const RoyaltyTooltip = ({ selectedMarket }: { selectedMarket: ListingMarket }) => {
+const MarketIcon = styled.img`
+  width: 16px;
+  height: 16px;
+  border-radius: 2px;
+  object-fit: cover;
+  outline: 1px solid ${({ theme }) => theme.backgroundInteractive};
+  margin-right: 8px;
+`
+
+const CollectionIcon = styled(MarketIcon)`
+  border-radius: 50%;
+`
+
+const FeePercent = styled(ThemedText.Caption)`
+  line-height: 16px;
+  color: ${({ theme }) => theme.textSecondary};
+  white-space: nowrap;
+`
+
+const MaxFeeContainer = styled(Row)`
```

Comment:
maybe add `cursor: default` so you don't get the text cursor

Reply:
the text cursor is the standard for all the elements in the grid so I think it's best to keep it for the fee as well to maintain consistency